### ISSUE TITLE
Ripme doesnt work with Imagefap

### ISSUE THREAD

**Issue by arachnidwoman** [RipMeApp/ripme#1783]
![Capture](https://user-images.githubusercontent.com/73803377/97820383-0cfd5080-1d02-11eb-9a0a-989650bc0f61.PNG)
* Ripme version:
* Java version: 
* Operating system: 

* Exact URL you were trying to rip when the problem occurred:
* Please include any additional information about how to reproduce the problem:

## Expected Behavior

Detail the expected behavior here.

## Actual Behavior

Detail the actual (incorrect) behavior here. You can post log snippets or attach log files to your issue report.

**Comment by doougal** [comment]
worked fine for me 
![image](https://user-images.githubusercontent.com/22046375/97830329-7052a700-1d31-11eb-805e-e6187b259448.png)
make sure you're sending in the gallery page, not a single photo, try other views etc

**Comment by AranciataExcess** [comment]
I've replicated it, try downloading 5+ albums in a row, it will prompt a captcha.

**Comment by ahmed-sys** [comment]
I have same issue too

**Comment by SIREN587** [comment]
Yep cant queue more than 5-6 items before getting that error, I will try with a list and report if it persists

**Comment by ringofyre** [comment]
Despite the new update  on 24/11/20 (thank you) it's just downloading thumbnails.
